src/modules/75-cf/pages/feature-flags/components/RenderFeatureFlag.tsx

```diff
@@ -0,0 +1,284 @@
+  numberOfEnvs,
+  gitSync,
+  toggleFeatureFlag,
+  governance,
+  cell: { row },
+  refetchFlags
+}) => {
+  const data = row.original
+  const [status, setStatus] = useState(isFeatureFlagOn(data))
+  const { getString } = useStrings()
+  const { showError } = useToaster()
+  const [flagNameTextSize, setFlagNameTextSize] = useState(300)
+  const ref = useRef<HTMLDivElement>(null)
+  const { activeEnvironment } = useActiveEnvironment()
+  const { handleError: handleGovernanceError, isGovernanceError } = governance
+
+  const [isSaveToggleModalOpen, setIsSaveToggleModalOpen] = useState(false)
+
+  const { gitSyncInitialValues, gitSyncValidationSchema } = gitSync.getGitSyncFormMeta(GIT_COMMIT_MESSAGES.TOGGLED_FLAG)
+
+  const [canToggle] = usePermission(
+    {
+      resource: {
+        resourceType: ResourceType.ENVIRONMENT,
+        resourceIdentifier: activeEnvironment
+      },
+      permissions: [PermissionIdentifier.TOGGLE_FF_FEATUREFLAG]
+    },
+    [activeEnvironment]
+  )
+
+  const handleFlagToggle = async (gitSyncFormValues?: GitSyncFormValues): Promise<void> => {
+    let gitDetails: GitDetails | undefined
+
+    if (gitSync.isAutoCommitEnabled) {
+      gitDetails = gitSyncInitialValues.gitDetails
+    } else if (gitSyncFormValues) {
+      gitDetails = gitSyncFormValues.gitDetails
+    }
+
+    try {
+      let response
+      if (status) {
+        response = await toggleFeatureFlag.off(data.identifier, gitDetails)
+      } else {
+        response = await toggleFeatureFlag.on(data.identifier, gitDetails)
+      }
+
+      if (isGovernanceError(response)) {
+        handleGovernanceError(response)
+      }
+
+      if (gitSyncFormValues?.autoCommit) {
+        gitSync.handleAutoCommit(gitSyncFormValues.autoCommit)
+      }
+
+      setStatus(!status)
+      refetchFlags()
+    } catch (error: any) {
+      if (error.status === GIT_SYNC_ERROR_CODE) {
+        gitSync.handleError(error.data as GitSyncErrorResponse)
+      } else {
+        if (isGovernanceError(error?.data)) {
+          handleGovernanceError(error.data)
+        } else {
+          showError(getErrorMessage(error), 0, 'cf.toggle.ff.status.error')
+        }
+      }
+    }
+  }
+
+  const NoEnvironmentWarningTooltip = (): ReactElement => (
+    <Popover interactionKind={PopoverInteractionKind.HOVER} popoverClassName={Classes.DARK}>
+      <Layout.Vertical padding="medium" className={css.tooltip}>
+        <Text font={{ weight: 'bold' }} color={Color.WHITE} padding={{ bottom: 'large' }}>
+          {getString('cf.noEnvironment.title')}
+        </Text>
+        <Text color={Color.GREY_200}>
+          <String stringID={'cf.noEnvironment.message'} useRichText />
+        </Text>
+      </Layout.Vertical>
+    </Popover>
+  )
+
+  const switchTooltip = (
+    <Container width={'350px'} padding="xxxlarge" className={css.switchTooltip}>
+      <Heading level={2} style={{ fontWeight: 600, fontSize: '24px', lineHeight: '32px', color: '#22222A' }}>
```

Comment:
this is probably legacy code?
```suggestion
      <Heading level={2} style={{ fontWeight: 600, fontSize: '24px', lineHeight: '32px', color: {Color.GREY_800} }}>
```